docs/source/examples/styling.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "Styling"

```

Comment:
Could you use a level 1 heading here? Like `# Styling`, to be consistent with the other notebooks https://github.com/poliastro/poliastro/blame/5702ec937721389394a7721e862c10de654c4ad7/docs/source/examples/Catch%20that%20asteroid!.ipynb#L7